src/core/hasVerifiedAttestations.ts

```diff
@@ -0,0 +1,34 @@
+import type { Address } from 'viem';
+import type { Chain } from 'viem';
+import { AttestationSchema } from './types';
+import { getAttestation } from './getAttestation';
+import { isChainSupported, getChainSchemasUids } from './attestation';
+
+/**
+ * Checks if the specified address has verified attestations for the given chain and expected schemas.
+ *
+ * @param chain - The blockchain to check for attestations.
+ * @param address - The address to check for attestations.
+ * @param expectedSchemas - An array of attestation schemas that are expected.
+ * @returns A promise that resolves to a boolean indicating whether the address has the expected attestations.
+ * @throws Will throw an error if the chain is not supported.
+ */
+export async function hasVerifiedAttestations(
+  chain: Chain,
+  address: Address,
+  expectedSchemas: AttestationSchema[] = [],
```

Comment:
`expectedSchemas` sounds odd. What exactly are we passing here?

Reply:
Oh I see we have those `'VERIFIED ACCOUNT' | 'VERIFIED COUNTRY'`, not sure I follow. 

Step 1, tell me the chain
Step 2, tell me an address, ok but who address? Customer address? unclear.
Step 3, tell me a schema, ok what schema? how do I know where those schemas are?

Reply:
`address` is probably fine, but we need good example in the docs on how this works and how and when should be used.